pandas/src/generate_code.py

```diff
@@ -1,3 +1,10 @@
+"""This file generates `generated.pyx` which is then included in `../algos.pyx`
+during building.  To regenerate `generated.pyx`, just run:
+
+    `python2 generate_code.py`.
+
```

Comment:
You sure?  Some systems now have `python` pointing to python3 (ex: Arch Linux) whereas I believe all systems recognize `python2`... and this file is not Python 3 compatible...